<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RSB6NC4" failures="0" tests="6" name="Default test" time="0.234" errors="0" timestamp="2022-09-20T15:26:38 IST">
  <testcase classname="extentSparkReportingV5.DemoTest" name="@BeforeTest beforeTest" time="0.198">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at org.apache.log4j.Logger.getLogger(Logger.java:40)
at freemarker.log._Log4jLoggerFactory.getLogger(_Log4jLoggerFactory.java:31)
at freemarker.log.Logger.getLogger(Logger.java:362)
at freemarker.template.Configuration.<clinit>(Configuration.java:151)
at com.aventstack.extentreports.reporter.AbstractFileReporter.loadTemplateModel(AbstractFileReporter.java:54)
at com.aventstack.extentreports.reporter.ExtentSparkReporter.start(ExtentSparkReporter.java:136)
at com.aventstack.extentreports.reporter.ExtentSparkReporter.access$000(ExtentSparkReporter.java:27)
at com.aventstack.extentreports.reporter.ExtentSparkReporter$1.onSubscribe(ExtentSparkReporter.java:116)
at io.reactivex.rxjava3.subjects.PublishSubject.subscribeActual(PublishSubject.java:133)
at io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13099)
at com.aventstack.extentreports.ReactiveSubject.attachReporter(ReactiveSubject.java:38)
at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:66)
at extentSparkReportingV5.DemoTest.beforeTest(DemoTest.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.apache.logging.log4j.LoggingException: log4j-slf4j-impl cannot be present with log4j-to-slf4j
at org.apache.logging.slf4j.Log4jLoggerFactory.validateContext(Log4jLoggerFactory.java:71)
at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:51)
at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:33)
at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getLogger(AbstractLoggerAdapter.java:53)
at org.apache.logging.slf4j.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:33)
at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:358)
at org.apache.logging.slf4j.SLF4JLoggerContext.getLogger(SLF4JLoggerContext.java:36)
at org.apache.log4j.Category.<init>(Category.java:176)
at org.apache.log4j.Category.<init>(Category.java:192)
at org.apache.log4j.Logger.<init>(Logger.java:57)
at org.apache.log4j.spi.RootLogger.<init>(RootLogger.java:39)
at org.apache.log4j.LogManager.<clinit>(LogManager.java:72)
... 37 more
]]>
    </failure>
  </testcase> <!-- @BeforeTest beforeTest -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="@AfterTest afterTest" time="-1.663667798831E9">
    <skipped/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f1" time="0.0">
    <skipped/>
  </testcase> <!-- f1 -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f2" time="0.0">
    <skipped/>
  </testcase> <!-- f2 -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f3" time="0.0">
    <skipped/>
  </testcase> <!-- f3 -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f4" time="0.0">
    <skipped/>
  </testcase> <!-- f4 -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f5" time="0.0">
    <skipped/>
  </testcase> <!-- f5 -->
  <testcase classname="extentSparkReportingV5.DemoTest" name="f6" time="0.0">
    <skipped/>
  </testcase> <!-- f6 -->
</testsuite> <!-- Default test -->
